workitem.go

```diff
@@ -98,7 +101,11 @@ func (c *WorkitemController) List(ctx *app.ListWorkitemContext) error {
 // Create runs the create action.
 func (c *WorkitemController) Create(ctx *app.CreateWorkitemContext) error {
 	return application.Transactional(c.db, func(appl application.Application) error {
-		wi, err := appl.WorkItems().Create(ctx.Context, ctx.Payload.Type, ctx.Payload.Fields)
+		currentUser, err := login.ContextIdentity(c.tokenManager, ctx)
```

Comment:
Maybe bring back a 'middleware'to set the tokenManager as part of the Context. Then ContextIdentity could get it from there.